src/app/controllers/applicationcontroller/applicationcontroller.go

```diff
@@ -83,10 +94,34 @@ func SaveApplicationDataHandler(w http.ResponseWriter, req *http.Request, app *m
 		helpers.SendError(http.StatusInternalServerError, friendlyErr, w)
 		return
 	}
+	go func() {
```

Comment:
I think this anonymous function should be removed.  The `go sendPublishEvent` below already does this.